packages/apps/transfer/src/pages/transfer/module-explorer/index.tsx

```diff
@@ -33,7 +35,22 @@ const GetCode: FC = () => {
   const [moduleChain, setModuleChain] = useState<number>(1);
   const [results, setResults] = useState<ModuleResult>({});
 
-  const networkdId = 'testnet04';
+  const { network } = useAppContext();
+
+  const chainNetwork: {
```

Comment:
I see this code more often, we should probably make this into a constant

Reply:
That would be great

Reply:
I created a separate task for this, no need to do it in this PR.